lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -2609,18 +2708,20 @@ void StmtEmitter::emitExpression(Value exp,
 
 /// Emit SystemVerilog attributes attached to the statement op as dialect
 /// attributes.
-void StmtEmitter::emitSVAttributes(ArrayAttr svAttrs, Operation *op) {
+void StmtEmitter::emitSVAttributes(Operation *op) {
```

Comment:
How is this different from the new function above?

Reply:
Slightly different. ExprEmitter doesn't emit indent and newline.